src/util.h

```diff
@@ -778,24 +779,16 @@ inline v8::MaybeLocal<v8::Value> ToV8Value(v8::Local<v8::Context> context,
         .Check();                                                              \
   } while (0)
 
-enum class Endianness { LITTLE, BIG };
-
-inline Endianness GetEndianness() {
-  // Constant-folded by the compiler.
-  const union {
-    uint8_t u8[2];
-    uint16_t u16;
-  } u = {{1, 0}};
-  return u.u16 == 1 ? Endianness::LITTLE : Endianness::BIG;
+constexpr inline bool IsLittleEndian() {
```

Comment:
Can this be `const` as well? (also IsBigEndian)?

```suggestion
constexpr inline bool IsLittleEndian() const {
```

Reply:
I don't think so? What intended effect are you hoping for?

Reply:
Is this blocking @anonrig? I am not quite sure what this would accomplish.

Reply:
I'd suggest if `const` is possible then it can be done in a separate PR. This change looks ready to go.